tokenization-example/readme.md

```diff
@@ -0,0 +1,44 @@
+# Merchant Server Example For Gopay Tokenization SDK
+
+## Overview
+
+This is the merchant server example for Gopay Tokenization SDK.
+
+In general the merchant server will pass all the request body and header to Partner API. The only change is to embed the `Server Key` that is base64 encoded into the request header as an authorization header.
+
+The endpoint served are for:
+
+* Account Linking
+* Account Status Enquiry
+* Create Transactions
+* Unlinking Account
+
+## Requirements
+
+Requirements to run this example server are:
+
+* Python 3.5+
+* Virtualenv
+* Valid Midtrans Server Key
+
+## Running
+
+* Setup virtual environment by running `$ virtualenv env`
+* Activate virtual environment `$ source env/bin/activate`
+* Install requirements `$ pip install -r requirements.txt`
+* Put your server key in the server key section in `app.py`
+* Run the server `$ python app.py`
+
```

Comment:
Maybe add:
The server will run on port `5000` by default (configurable).

Reply:
done